framework/include/cppmicroservices/detail/ServiceTracker.hpp

```diff
@@ -151,9 +151,18 @@ namespace cppmicroservices
     void
     ServiceTracker<S, T>::Close()
     {
+
+        /* Swap default ListenerToken with d-listenerToken required to prevent data race in call to
```

Comment:
nit-pick: typo in `d-listenerToken`

The comment is a little confusing, specifically the use of the word `default`. Did you mean `default constructored`?  Also, this comment doesn't explain the "why", which is more imporatnt.